cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -131,18 +136,30 @@ class LeadImpl extends ServerImpl with Lead with Logging {
 
       if (bootProperties.getProperty(Constant.ENABLE_ZEPPELIN_INTERPRETER,
         "false").equalsIgnoreCase("true")) {
-        /**
-         * This will initialize the zeppelin repl interpreter.
-         * This should be done before spark context is created as zeppelin
-         * interpreter will set some properties for classloader for repl
-         * which needs to be specified while creating sparkcontext in lead
-         */
-        val props: Properties = ZeppelinIntpUtil.initializeZeppelinReplAndGetConfig()
-        props.asScala.foreach(kv => conf.set(kv._1, kv._2))
-      }
-      logInfo("About to initialize SparkContext with SparkConf=" + conf.toDebugString)
 
+        try {
+
+          var zeppelinIntpUtilClass = Class.forName("org.apache.zeppelin.interpreter.ZeppelinIntpUtil")
+
+          /**
+           * This will initialize the zeppelin repl interpreter.
+           * This should be done before spark context is created as zeppelin
+           * interpreter will set some properties for classloader for repl
+           * which needs to be specified while creating sparkcontext in lead
+           */
+          logInfo("About to initialize SparkContext with SparkConf")
+          val method: Method = zeppelinIntpUtilClass.getMethod("initializeZeppelinReplAndGetConfig")
+          val obj: Object = method.invoke(null)
+          val props: Properties = obj.asInstanceOf[Properties]
+          props.asScala.foreach(kv => conf.set(kv._1, kv._2))
+        } catch {
+          /* [Sachin] So we need to log warning that
```

Comment:
I think exiting will make sense. What is the default value of ENABLE_ZEPPELIN_INTERPRETER ?